### ISSUE TITLE
Accessibility: The quantity increment/decrement input loses focus after changing the number of products.

### ISSUE THREAD

**Issue by msiewierska** [shopware/platform#14015]
### Shopware Version

6.7.3.1

### Affected area / extension

Platform(Default)

### Actual behaviour

On the cart page 2.4.3 WCAG criteria (level A) is violated: focus after quantity update should stay on quantity selector (currently is moved to the top of the page).

In the minicart focus is moved to the top of the offcanvas

### Expected behaviour

 Focus after quantity update should stay on quantity input.

### How to reproduce

Go to the cart page or open minicart with at least one product. Update amount of product with keyboard.
